target/linux/ramips/image/rt305x.mk

```diff
@@ -1225,6 +1225,16 @@ define Device/zyxel_keenetic-start
 endef
 TARGET_DEVICES += zyxel_keenetic-start
 
+define Device/zyxel_keenetic-4g-b
```

Comment:
`zyxel_keenetic-4g-b` should go above `zyxel_keenetic-lite-b`.

Reply:
moved